package.json

```diff
@@ -18,6 +18,10 @@
     "./lib/compat": {
       "import": "./lib/compat.js",
       "require": "./lib/compat.cjs"
+    },
+    "./lib/helpers": {
```

Comment:
We don't need this. You can reexport all the helpers function in `src/session.ts` so we can access it like:

```js
import { isNew, isTouched, isDestroyed } from 'next-session';
```